integrated_channels/integrated_channel/models.py

```diff
@@ -881,41 +881,58 @@ class Meta:
     resolved = models.BooleanField(default=False)
 
 
-class IntegratedChannelAPIRequestLogs(TimeStampedModel):
+class BaseIntegratedChannelAPIRequestLogs(TimeStampedModel):
```

Comment:
[Question] Curious to know the intention behind creating this abstract class.

Reply:
So In my previous PR where I extended/inherited Cornerstone model from IntergratedChannelApiRequestLogs model - when I looked at the table, it basically created two tables(first one with all columns and second one with ForeignKey relation with the first one). 
But I wanted two different tables so had to create a base abstract class; which we can remove now if we want but just left it as it is in case if needed in future.

Reply:
Removed!